README.md

```diff
@@ -145,6 +147,30 @@ This integer specifies the LUKS key size (in bits).
 ##### `encryption_luks_version`
 This integer specifies the LUKS version to use.
 
+##### `deduplication`
+This specifies whether or not the Virtual Data Optimizer (VDO) will be used.
+When set, duplicate data stored on storage volume will be
+deduplicated resulting in more storage capacity.
+Can be used together with `compression` and `vdo_pool_size`.
+Volume has to be part of the `storage_pool`.
+Limit one VDO `storage_volume` per `storage_pool`.
+Underlying volume has to be at least 9 GB (bare minimum is around 5 GiB).
+
+##### `compression`
+This specifies whether or not the Virtual Data Optimizer (VDO) will be used.
```

Comment:
Same thing here. (lvm-only, how will we handle it for non-lvm?)

Reply:
Fixed.